cloud/blockstore/apps/client/lib/create_volume.cpp

```diff
@@ -179,6 +182,19 @@ class TCreateVolumeCommand final
             return true;
         }
 
+        if (JsonOutput){
+            // We don't use result.PrintJSON(), because
+            // TError.PrintJSON() writes only code, and
+            // it is more reliable to use formatted error in
+            // tests and scripts.
```

Comment:
можно в три строчки:
```
            // We don't use result.PrintJSON(), because TError.PrintJSON()
            // writes only code, and it is more reliable to use formatted 
            // error in tests and scripts.
```